CONTRIBUTING.md

```diff
@@ -329,6 +329,69 @@ If you already have the `refinebio-examples` docker image:
+
+After you've prepared the new or altered material for this repository, you can file a pull request to the `staging` branch (the default branch for this repository).
+The `staging` branch does not feed into the published, user-facing material so you can feel free to make your pull requests as iterative and incremental as it is useful.
+Use `squash and merge` for these merges.
+`squash and merge` will be helpful for us if we need to "cherry pick" commits if particular project or example is not yet ready for user-facing but other material is.
+
+#### Make it live: staged changes merged to master
+
+Once a set of changes that are merged to `staging` are ready to be published and made "live", you can file a merge to `master` pull request.
+These types of PRs should only involve well-polished and ready for the public material.
+
+**Scenario 1: All changes from staging should be made live**  
+
+If you are certain that all changes in `staging` should be carried over to `master`, first create a new branch from the most up-to-date `staging` branch.
+Now you can use the new branch to create the pull request to master as you would normally do.
+This allows you can resolve any merge conflicts in your new branch so we don't do a bad thing by committing conflict resolutions directly to `staging`.
+
+**Scenario 2: Only some changes from staging should be made live**  
+
+If only some changes in `staging` are ready to be public facing (and there isn't great timing for when the other changes will be ready), then you can use cherry picking to only bring over the changes ready to be live.
+This time, create a new branch from the most up-to-date `master` branch which you will use for merging.
+
+_How to cherry pick commits_
+Checkout your new branch.
+For each commit that needs to be made live, add it to your new branch by using `git cherry-pick <commit_id>`.
+Or in GitKraken, you can right click on the commit and choose `Cherry pick commit`.
+
+_Tips for getting commits ids_  
+GitKraken shows commits, but sometimes I find it hard to follow which commits belong to which branch.
+If you checkout `staging` and use `git log` you can see all the most recent commits for the `staging` branch.
+If you want to save it to a file for easy browsing and copy/pasting you can use this command `git log --pretty=format:'%h was %an, %ar, message: %s' > git.log`
+
+#### Make it live, but quicker: direct merges to master, hotfixes
+
+In (hopefully rare) scenarios where something that has already been published is noted to be broken and should be addressed quickly, [hotfix branch](https://nvie.com/posts/a-successful-git-branching-model/#hotfix-branches) pull requests are allowed.
+These PRs should only be fairly small PRs and not anything that would require intense review.
+This more for situations where "this is broken and here's a fix".
+
+After the change has been sent to `master`, you should also make sure the change still ends up in `staging`.
```

Comment:
I don't like how vague this part is, not sure how to make this less annoying but still ultimately ensure that `staging` and `master` don't get too far out of date.

Reply:
How about something like "Once your PR  to `master` is approved, file a second PR from the branch to `staging`"